AbletonMCP_Remote_Script/handlers/audio.py

```
@@ -0,0 +1,240 @@
+                if hasattr(sample, "length"):
+                    analysis["audio_properties"]["sample_length"] = sample.length
+                    if hasattr(sample, "sample_rate") and sample.sample_rate > 0:
+                        analysis["audio_properties"]["duration_seconds"] = sample.length / sample.sample_rate
+                        analysis["audio_properties"]["sample_rate"] = sample.sample_rate
+                if hasattr(sample, "bit_depth"):
+                    analysis["audio_properties"]["bit_depth"] = sample.bit_depth
+                if hasattr(sample, "channels"):
+                    analysis["audio_properties"]["channels"] = sample.channels
+                    analysis["audio_properties"]["is_stereo"] = sample.channels == 2
+            except Exception:
+                pass
```

Ruff (0.15.0)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)